package.json

```diff
@@ -22,6 +22,7 @@
     "cross-spawn": "^2.2.3",
     "gulp-mocha": "^2.2.0",
     "lodash": "^4.11.1",
+    "spectron": "^3.0.0",
```

Comment:
it seems that `spectron` is used in only test code. Can you move it to devDependencies?